node/overseer/overseer-gen/proc-macro/src/impl_builder.rs

```diff
@@ -351,11 +513,11 @@ pub(crate) fn impl_builder(info: &OverseerInfo) -> proc_macro2::TokenStream {
 					let (signal_tx, signal_rx) = #support_crate ::metered::channel(SIGNAL_CHANNEL_CAPACITY);
 
 					// Generate subsystem name based on overseer field name.
-					let mut subsystem_string = String::from(stringify!(#subsystem_name));
+					let subsystem_string = String::from(stringify!(#subsystem_name));
 					// Convert owned `snake case` string to a `kebab case` static str.
 					let subsystem_static_str = Box::leak(subsystem_string.replace("_", "-").into_boxed_str());
```

Comment:
note to myself: this could be extracted as generated `&'static str` rather than the adhoc string generation.